Koans/AboutLinq.cs

```diff
@@ -0,0 +1,79 @@
+﻿using DotNetCoreKoans.Engine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace DotNetCoreKoans.Koans
+{
+    public class AboutLinq : Koan
+    {
+        /*
+         *LINQ (Language Integrated Query).
+         * LINQ provides language-level querying capabilities and a higher-order function API to C# and VB as a way to write expressive, declarative code.
+         * All LINQ query operations consist of three distinct actions:
+
+            1. Obtain the data source.
+
+            2. Create the query.
+
+            3. Execute the query.
+         */
+
+        [Step(1)]
+        public void FilterArrayData()
+        {
+            //This sample uses where to find all elements of an array less than 5.
+
+            // The Three Parts of a LINQ Query:
+
+            //  1. Data source.
+            int[] numbers = { 5, 1, 9, 8, 6, 7 };
+
+            // 2. Query creation.
+            var lowNums =
+                from n in numbers
+                where n < 5
+                select n;
+
+            // 3. Query execution.
+            Assert.Equal(FILL_ME_IN, lowNums.Count());
```

Comment:
Instead of asserting the count, I would change this to be the expected values in the result.

Reply:
You're right it makes more sense. I'm on it.